api/src/pcapi/core/object_storage/backends/gcp.py

```diff
@@ -36,6 +37,8 @@ def delete_public_object(self, folder: str, object_id: str) -> None:
             bucket = self.get_gcp_storage_client_bucket()
             gcp_cloud_blob = bucket.blob(storage_path)
             gcp_cloud_blob.delete()
+        except NotFound as not_found_exc:
+            logger.info("File not found on GCP bucket: %s", not_found_exc)
```

Comment:
(détail / suggestion) Il n'y en a pas pour l'instant, mais il y aura peut-être plus tard d'autres messages de log qui parleront de "File not found". On pourrait rendre celui-ci plus spécifique pour indiquer qu'ici c'est pour une suppression, comme suit : 

```suggestion
            logger.info("File not found on deletion on GCP bucket: %s", storage_path)
```

... ou, un peu plus verbeux : 

```suggestion
            logger.info("Tried to cancel unknown object on GCP bucket: %s", storage_path)
```